docs/static/ls-ls-config.asciidoc

```diff
@@ -0,0 +1,105 @@
+[[ls-to-ls]]
+=== Configuring Logstash-to-Logstash Communication
+
+In a complex network environment, you may need two Logstash machines talking to each other. That is, the output from the first Logstash machine becomes the input for the second Logstash machine. You may need this configuration if the data path crosses network or firewall boundaries, for example.
```

Comment:
I would add to this first section a succinct description of the solution, for those who don't need a tutorial. "The recommended way for one Logstash to communicate with another is by connecting the Lumberjack output to the Beats input", which links to the output and input respectively.

Reply:
Can we globally replace 'first' and 'second' with 'upstream' and 'downstream', that more clearly documents how these Logstash instances relate to one another.

I think we can generally reword this to say something along the lines of.

"To connect one Logstash to another we use the Lumberjack protocol, which is spoken by the lumberjack output and the beats input. To use it, setup the downstream Logstash instance with the Beats input to listen for incoming Lumberjack connections, then, setup the upstream Logstash instance with the Lumbjerjack output to connect to it."